`Apply terraform` job errors out due to duplicated model

### Bug Description

When the `terraform-checks' Apply terraform` runs without a `model` input, it fails with the following message:

```
ERROR cannot create model "testing": a namespace called "testing" already exists on this k8s cluster. Please pick a different model name.
```

This is caused by:

* The `terraform-apply` workflow [creates](https://github.com/canonical/charmed-kubeflow-workflows/blob/main/.github/workflows/terraform-apply.yaml#L43C1-L44C48) a juju model that defaults to `testing` 
* At the same time, the workflow calls `charmed-kubernetes/actions-operator` to setup the testing environment on a step before. This action already [creates a model](https://github.com/search?q=repo%3Acharmed-kubernetes%2Factions-operator%20add-model&type=code) called `testing`
* Because of the above, the workflow fails to create a model that already exists

### To Reproduce

Check the CI that fails, for example [this](https://github.com/canonical/dex-auth-operator/actions/runs/10950443420/job/30405675599#step:5:7)

### Environment

CI for Terraform

### Relevant Log Output

```shell
Run juju add-model testing
  juju add-model testing
  shell: /usr/bin/bash -e {0}
  env:
    CONTROLLER_NAME: github-pr-103aa-microk8s
ERROR cannot create model "testing": a namespace called "testing" already exists on this k8s cluster. Please pick a different model name.
Error: Process completed with exit code 1.
```


### Workaround

Define a model name different than `testing` when calling the wf:

```
  terraform-checks:
    name: Terraform
    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
    with:
      charm-path: .
      model: kubeflow
```

### Potential solution

Change the default model name of the `terraform-apply` workflow from `testing` to `tf-testing`

--- a/.github/workflows/terraform-apply.yaml
+++ b/.github/workflows/terraform-apply.yaml
@@ -8,9 +8,8 @@ on:
     inputs:
       model:
         description: The model that the charm is deployed on. Defaults to `testing`.
-        required: false
+        required: true
         type: string
-        default: testing
       channel:
         description: The channel that the charm is deployed from. Defaults to `latest/stable`.
         required: false

--- a/.github/workflows/terraform-apply.yaml
+++ b/.github/workflows/terraform-apply.yaml
@@ -8,8 +8,9 @@ on:
     inputs:
       model:
         description: The model that the charm is deployed on. Defaults to `testing`.
-        required: true
+        required: false
         type: string
+        default: tf-testing
       channel:
         description: The channel that the charm is deployed from. Defaults to `latest/stable`.
         required: false

--- a/.github/workflows/terraform-checks.yaml
+++ b/.github/workflows/terraform-checks.yaml
@@ -11,7 +11,7 @@ on:
         description: The model that the charm is deployed on. Defaults to `testing`.
         required: false
         type: string
-        default: testing
+        default: tf-testing
       channel:
         description: The channel that the charm is deployed from. Defaults to `latest/stable`.
         required: false

